src/components/BannerConductor/StateMachine/states/ClosedState.ts

```diff
@@ -1,11 +1,19 @@
 import { BannerState } from '@src/components/BannerConductor/StateMachine/states/BannerState';
 import { BannerStates } from '@src/components/BannerConductor/StateMachine/BannerStates';
+import { Page } from '@src/page/Page';
 
 export class ClosedState extends BannerState {
 	stateName: BannerStates = BannerStates.Closed;
+	page: Page;
+
+	constructor( page: Page ) {
+		super();
+		this.page = page;
+	}
 
 	enter(): Promise<any> {
-		// fire events
+		this.page.unsetAnimated();
```

Comment:
We should add a method to the page called `removeBanner()` or `hideBanner()` that also stops resize event listeners

Reply:
Oh, and calls `mw.centralNotice.hideBanner` thorought the MediaWiki object